lock/src/main/java/com/auth0/android/lock/enums/UsernameStyle.java

```diff
@@ -30,5 +30,5 @@
 public enum UsernameStyle {
     EMAIL,
     USERNAME,
-    DEFAULT
+    USERNAME_OR_EMAIL
```

Comment:
Let's stick with `DEFAULT`, or a better name, since it can be user or email or user and email depending on the connection configuration. Other option is to have `null` mean default